app/assets/stylesheets/_honeycrisp-compact.scss

```diff
@@ -301,6 +301,12 @@
 
   .client-navigation {
     @include no-bleed();
+    .security-wrapper{
```

Comment:
the tiniest nitpick ever is that there should be a space between the class name and the bracket! 🦐